### ISSUE TITLE
Too long texts in ReadTheDocs

### ISSUE THREAD

**Issue by matthias-ronge** [kitodo/kitodo-production#4080]
Our menu texts are too long, so the control line breaks into two lines and covers the upper part of the help page:

![Screenshot of Kitodo.Production help](https://user-images.githubusercontent.com/3040657/97959611-05948080-1db0-11eb-8591-c0db988f5d8e.png)

With short texts, it looks right:

![Screenshot of correct example](https://user-images.githubusercontent.com/3040657/97959617-07f6da80-1db0-11eb-8760-796702dcb5bd.png)

**Comment by Erikmitk** [comment]
My preferred solution would be to change/adjust the theme. https://www.mkdocs.org/user-guide/styling-your-docs/

**Comment by Erikmitk** [comment]
Actually… it looks like the ReadTheDocs (RTD) infrastructure is at fault. The long values aren't even supposed to be there. README has no place in the navigation. Something fishy in the build process is going on.

My fork builds cleanly in RTD. The difference is that I have the Python Interpreter for building set to 3.x. When I try to change the official documentation to the same settings it fails to build. I suspect an outdated mkdocs dependency due to the error message. I tried [wiping the build environment](https://docs.readthedocs.io/en/stable/guides/wipe-environment.html) but that didn't change a thing. 😞

**Comment by Erikmitk** [comment]
https://docs.readthedocs.io/en/stable/intro/getting-started-with-mkdocs.html
> We strongly recommend to pin the MkDocs version used for your project to build the docs to avoid potential future incompatibilities.

We might have to add a configuration.

(Although it's super weird that my fork builds the documentation clean out of the box.)

**Comment by bertsky** [comment]
> My fork builds cleanly in RTD.

@Erikmitk, what do you mean by that? Do you have a readthedocs.io deployment for your fork (where?), or is it just your local `mkdocs serve -t readthedocs` (or just `mkdocs serve`) that looks correct?

IMO the issue here is actually much broader: currently, in the RTD deployment, none of the images in the user documentation resolve correctly. I attempted #5221 to solve that (by prefixing all references with `/docs`), only to find out the image paths were correct, but the deployment was not.

So this is quite urgent, actually. (Also, running mkdocs should be **documented** and probably put into the CI...)

**Comment by Erikmitk** [comment]
@bertsky This is quite an old issue but I just checked if it still persists. I have access to two projects on RTD:

<img width="707" alt="Screen Shot 2022-07-06 at 23 25 29" src="https://user-images.githubusercontent.com/180686/177646279-e437386c-95ab-45cc-a46f-3c1d74da2347.png">

Both can be built but [*this* official .Production repo](https://github.com/kitodo/kitodo-production) renders a broken [documentation](https://kitodo-production.readthedocs.io/en/master/).

[My identical fork](https://github.com/Erikmitk/kitodo-production.git ) renders [a readable version](https://kitodoproduction-dev.readthedocs.io/en/master/).

I gave up on this in November 2020 because it seemed like an issue on the RTD platform itself where the build process was somehow glitchy. That's when I added the comment about adding a RTD config to the project. There were no differences in the settings in both projects on RTD and the source code is identical. Even locally `mkdocs serve -t readthedocs` worked for me as well. I still have no idea why this happens.

**Comment by bertsky** [comment]
> [My identical fork](https://github.com/Erikmitk/kitodo-production.git) renders [a readable version](https://kitodoproduction-dev.readthedocs.io/en/master/).

Interesting. However, did you notice that even in your deployment, within the user section, none of the images resolve correctly? (They come with absolute links like `/user/pictures/ProduktionsvorlageLaufzettelAuswaehlen.png`, lacking either `/en/master` or `../../..` as a prefix.)

In a local deployment (with latest `mkdocs==1.3.0` for Python 3.7), these image links are all correct.

I just registered with RTD myself and set up a build for my GH fork of Production (setting `master / mkdocs / English / Java`). I ran into [this build error](https://readthedocs.org/projects/kitodoproduction-bertsky/builds/17374422/). To me this seems to be caused by readthedocs.org configuring `mkdocs<1.1`. The default config does set `CPython 3.X`, so at least your concern regarding Python 2 vs 3 seems to be gone. It offers selecting a requirements.txt or conf.py from the repo, which obviously (this being a Java project) we don't have. So _something_ within RTD drags in that bogus `mkdocs<1.1` requirement...

(Unfortunately, the [RTD docs on Github integration](https://docs.readthedocs.io/en/stable/integrations.html) does not have to say anything about mkdocs. And on the Github side, Settings → Webhooks → RTD, I cannot configure anything related to the generator at all.)

**Comment by bertsky** [comment]
> To me this seems to be caused by readthedocs.org configuring `mkdocs<1.1`. The default config does set `CPython 3.X`, so at least your concern regarding Python 2 vs 3 seems to be gone

Ha! It _does_ in fact revolve around that problem, though: Comparing [my build log](https://readthedocs.org/projects/kitodoproduction-bertsky/builds/17374422/) to [your build log](https://readthedocs.org/projects/kitodoproduction-dev/builds/17373829/), one immediately notices:
- yours: Python 2.7, installs `... mkdocs`
- mine: Python 3.7, installs `... mkdocs<1.1`

So you _were_ right above.

I believe [this existing issue](https://github.com/readthedocs/readthedocs.org/issues/9081) explains the RTD side. IIUC, all we have to do is _amend_ our existing mkdocs.yml with a statement on the `mkdocs` version.

**Comment by bertsky** [comment]
Spoiler: I did [add a docs/requirements.txt](https://github.com/bertsky/kitodo-production/commit/18be3f0adbf97ff8162fccca1174d73a7f45289b), and it does repair the [Python 3 build of RTD](https://readthedocs.org/projects/kitodoproduction-bertsky/builds/17374731/). 

But the [images are still broken](https://kitodoproduction-bertsky.readthedocs.io/en/latest/user/introduction/productiontemplate/#beispiel).

So the culprit must be within the difference between locally running mkdocs and having RTD do it.

**Comment by bertsky** [comment]
> Yesterday I checked this [architecture section](https://kitodoproduction-dev.readthedocs.io/en/master/developer/architecture/) and all the images work perfectly fine!!

Yes, but in that section the images are actually either remote references or just filenames (i.e. relative to the Md directory).

> It looks like this is broken at the source since the images in the user section don't even work for me here in the repo:

Yes, in the user section, all images are in `/docs/users/pictures/...`, but the references use `/users/pictures/...` for some reason.

> So the images (paths, I guess) in the repo need to be fixed and a `requirements.txt` for the docs to get the same repeatable build process.

In #5221 I tried with additional `/docs` prefix, and that works on the Github website, but neither the locally or the RTD deployed mkdocs will work with these paths. (I did not try with `../../..` though.)

I believe there must be some setting (that likely has changed in recent mkdocs versions) for the reference of absolute links. So IMO we should fix the configuration, not the Markdown files.
